components/molecules/DSFooter.js

```diff
@@ -0,0 +1,13 @@
+import { Footer } from '@dts-stn/decd-design-system'
+
+export default function DSFooter() {
+  return (
+    <Footer
```

Comment:
Doesn't look like we need a component to hold a component

Reply:
nope... lol; something to solve later?